source/iterable.js

```diff
@@ -48,15 +48,14 @@ export const combineAsyncIterators = async function * (...iterators) {
 			}
 		}
 	} finally {
-		await Promise.all(iterators.map(iterator => iterator.return?.()));
+		await Promise.all(iterators.map(iterator => iterator.return()));
 	}
 };
 
 const getNext = async iterator => {
 	try {
 		return await iterator.next();
 	} catch (error) {
-		await iterator.throw?.(error);
-		throw error;
```

Comment:
`iterator` is the `lineIterator` created for either `stdout` or `stderr`. This function combines both. If an error is thrown in the `for` loop, `iterator.throw()` for both `stdout` and `stderr` is called, then the original error is propagated.

The code works for any type of `iterator`. But in practice, we know that the `iterator` is always created by a generator `function * () {}`. That type of iterator: 
  - Always has a `return()` and `throw()` functions (which are otherwise optional for other iterators) 
  - Always has a `throw()` that re-throws the `error` passed as argument. Therefore, `throw error` is never reached.

So this PR does a small adjustment to remove a few bytes, by optimizing the fact that we know that `iterator` is coming from a generator function.